Apps/Medication/src/Delegates/SalesforceDelegate.cs

```diff
@@ -113,7 +119,7 @@ public async Task<RequestResult<IList<MedicationRequest>>> GetMedicationRequests
                             if (replyWrapper != null)
                             {
                                 retVal.ResultStatus = ResultType.Success;
-                                retVal.ResourcePayload = replyWrapper.ToHGModels();
+                                retVal.ResourcePayload = this.autoMapper.Map<IEnumerable<SpecialAuthorityRequest>, IEnumerable<MedicationRequest>>(replyWrapper.Items).ToList();
```

Comment:
I think the code can be simplified by mapping directly to an IList<MedicationRequest> instead of calling ToList().